snap/info.go

```diff
@@ -236,6 +238,10 @@ func (app *AppInfo) SecurityTag() string {
 	return fmt.Sprintf("snap.%s.%s", app.Snap.Name(), app.Name)
 }
 
+func (app *AppInfo) EnvironmentFilePath() string {
```

Comment:
`FilePath` is actually nicer than just `Path` or just `File`, as it's clear and prevents the conflict with the concept of `os.File`. We should change the other (already inconsistent) ones to that in the future.